### ISSUE TITLE
Setup improvement idea (unclear interactive CLI tool configuration)

### ISSUE THREAD

**Issue by mrlubos** [sofn-xyz/mailing#74]
When running `mailing` for the first time, the interactive tool asks the following question:

```
? Where should we put your emails? › ./emails/
```

This doesn't make it clear that the only allowed values here are `./emails` and `./src/emails`. This should either be documented or refactored to truly allow custom values. Answering this question with `./src/mailing` will not work with the preview and there's currently no way to fix this.

**Comment by psugihara** [comment]
Ah great catch, thank you. I think we should add a key to the package.json with some config to store the file location...

```
{
  ...
  "mailing": {
    emailsDir: "src/mailing"
  }
}
```

**Comment by mrlubos** [comment]
@psugihara That sounds good! Personally, I'd prefer a separate config file or an option to use one instead of `package.json`

**Comment by psugihara** [comment]
yeah I like that too, mailing.config.json

**Comment by alexfarrill** [comment]
fixed by #116
